esx_service/cli/vmdkops_admin.xml

```diff
@@ -0,0 +1,571 @@
+<?xml version="1.0"?>
+<plugin
+    xmlns="http://www.vmware.com/Products/ESX/6.0/esxcli/">
+    <!-- esxcli extension syntax version -->
+    <version>1.0.0</version>
+    <namespaces>
+        <namespace path="vdvs">
```

Comment:
As suggested "storage containers volumes ..." seems a good option. I'd suggest that all  options would be under "storage containers volumes". Because "storage containers policy" would be misleading as we don't deal with containers as such - only container volumes.